docs/datadog_ktx_module.md

```diff
@@ -0,0 +1,49 @@
+# Datadog Kotlin Extensions package
```

Comment:
```# Kotlin Extensions```

Also, instead of `datadog_ktx_module.md` rename this file `kotlin_extensions.md`